doc/installation.md

```diff
@@ -59,6 +59,25 @@ Use `pip` to install the source code in "editable" mode. This means that Python
 
 You'll find that the `capa.exe` (Windows) or `capa` (Linux/MacOS) executables in your path now invoke the capa binary from this directory.
 
+#### Development
+
+##### venv [optional]
+
+For development, we recommend to use [venv](https://docs.python.org/3/tutorial/venv.html). It allows you to create a virtual environment: a self-contained directory tree that contains a Python installation for a particular version of Python, plus a number of additional packages. This approach avoids conflicts between the requirements of different applications on your computer. It also ensures that you don't overlook to add a new requirement to `setup.up` using a library already installed on your system.
+
+To create an environment (in the parent directory, to avoid commiting it by accident or messing with the linters), run:
+`$ python3 -m venv ../capa-env`
```

Comment:
Other options more developers seem to use are `<project>/.venv` or `<project>/.env`, but these are only recommendations here.

Reply:
I took this from the official documentation: https://docs.python.org/3/library/venv.html

Reply:
Right, I was only referring to the location of where to create the `venv` (outside the project dir vs. inside).

Reply:
aah! that is a good idea 👍  we could add those to `.gitignore`

Reply:
the problem is still that we have to remember excluding then from isort and black. I have just remember that was the reason to move it outside. I tend to think that it is easier to put it outside. Any other opinions?

Reply:
i keep mine outside to avoid .gitignore other similar issues.

i dont think we should spend too much time explaining to developers how to configure their systems, so if putting it outside makes this easier, my vote is for that.